app/Config/Schema/sessions.php

```diff
@@ -24,34 +25,35 @@
  * cake schema run create Sessions
  *
  */
-class SessionsSchema extends CakeSchema {
-
-	public $name = 'Sessions';
-
-/**
- * Before callback.
- *
- * @param array $event Schema object properties
- * @return bool Should process continue
- */
-	public function before($event = array()) {
-		return true;
-	}
+class SessionsSchema extends CakeSchema
+{
+    public $name = 'Sessions';
 
-/**
- * After callback.
- *
- * @param array $event Schema object properties
- * @return void
- */
-	public function after($event = array()) {
-	}
+    /**
+     * Before callback.
+     *
+     * @param array $event Schema object properties
+     *
+     * @return bool Should process continue
+     */
+    public function before($event = array())
+    {
+        return true;
+    }
 
-	public $cake_sessions = array(
-		'id' => array('type' => 'string', 'null' => false, 'key' => 'primary'),
-		'data' => array('type' => 'text', 'null' => true, 'default' => null),
-		'expires' => array('type' => 'integer', 'null' => true, 'default' => null),
-		'indexes' => array('PRIMARY' => array('column' => 'id', 'unique' => 1))
-	);
+    /**
+     * After callback.
+     *
+     * @param array $event Schema object properties
+     */
+    public function after($event = array())
+    {
+    }
 
+    public $cake_sessions = array(
+        'id' => array('type' => 'string', 'null' => false, 'key' => 'primary'),
+        'data' => array('type' => 'text', 'null' => true, 'default' => null),
+        'expires' => array('type' => 'integer', 'null' => true, 'default' => null),
+        'indexes' => array('PRIMARY' => array('column' => 'id', 'unique' => 1)),
```

Comment:
__[staging][code_sniffer]__
* Array key not aligned correctly; expected 29 spaces but found 8